### PR TITLE
kuba--.md

### PR DESCRIPTION
**Pull request by kuba--**

### CODE REVIEW

**Comment by smola** [comment]
@kuba-- You have to sign-off the commit (`git commit -s`), since we have the [DCO check](https://github.com/src-d/guide/blob/956c0d0d240abc8e5dd37df9fdb8bb8ce34768fe/engineering/licensing.md) enabled for the organization.

File: talent/sourcerers/kuba--.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- If you could have a coffee with any person in the world, who would it be?
+  - Actually I would more appriciate a coffee with any of my old friends which I haven't seen in person for years.
+- Would you like to share with us a story about yourself?
+  - Too long; Don't read...
+- What food do you like and dislike?
+  - I like almost any Itallian food, but I can't handle _sea food_.
+- What is your favorite quote?
+  - Maybe this one from _Ghost in the Shell_ movie:
+    ```
+    "If our Gods and our hopes are nothing but scientific phenomena, then let us admit it must be said that our love is scientific as well."
```
Comment (eiso) [inline talent/sourcerers/kuba--.md]: Amazing quote, hadn't heard it before.

File: talent/sourcerers/kuba--.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Kuba Podgórski
+
+**General**
+- What's your name? How do your friends call you?
+  - The official name is Jakub, but as a short version/nickname _Kuba_ is commonly used. Also all friends call me _Kuba_.
+- When were you born?
+  - 0x1A Feb 0x7BB
+- Where are you from? Where have you lived?
+  - I'm originally from Lublin (Poland), but I also lived for couple years in San Francisco (US) and now in Kraków (Poland).
```
Comment (campoy) [inline talent/sourcerers/kuba--.md]: "for a couple of years"

File: talent/sourcerers/kuba--.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- What languages (natural and programming ones) do you know?
+  - I speak Polish (native) and English. Regarding programming languages, I had a chance to touch almost everything from the main stream and some academia stuff (like Prolog and Lisp), but personally my preferences are _C_, _Go_, _Prolog_.
+- Would like to share your Twitter, Instagram or Medium or any other social media account?
+  - I do not use any social media (long time ago I was active on G+, posting mostly my photos)
+
+**Work**
+- What are you going to be doing at source{d}?
+  - I'm a Software Engineer at _Data Retrieval_ team, what means that I mainly work with different _distributted storage systems_.
```
Comment (campoy) [inline talent/sourcerers/kuba--.md]: "what means that I" -> "which means I"

File: talent/sourcerers/kuba--.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+- Why did you join source{d}?
+  - Great team focused on knowledge; Open source; It also matches my tech. preferences.
```
Comment (campoy) [inline talent/sourcerers/kuba--.md]: "tech." -> tech

File: talent/sourcerers/kuba--.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+- Where will you work from?
+  - Most often I work remotely from Kraków, Poland.
+- If you could only save one programming language, which one would it be?
+  - I think it would be _C_ as a good starting point to _reinvnent a wheel_.
```
Comment (campoy) [inline talent/sourcerers/kuba--.md]: reinvent is misspelled, also the expression is "reinvent *the* wheel"

File: talent/sourcerers/kuba--.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+      - Tail Recursion, ...knock, knock
+      ```
+
+      ```
+      - Knock, knock
+      - Who's there?
+      - See
+      - See, who?
+      - See food
+      ```
```
Comment (campoy) [inline talent/sourcerers/kuba--.md]: actually LOLed 😆

File: talent/sourcerers/kuba--.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+- If you could have a coffee with any person in the world, who would it be?
+  - Actually I would more appriciate a coffee with any of my old friends which I haven't seen in person for years.
```
Comment (campoy) [inline talent/sourcerers/kuba--.md]: appriciate -> appreciate

File: talent/sourcerers/kuba--.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+- Would you like to share with us a story about yourself?
+  - Too long; Don't read...
+- What food do you like and dislike?
+  - I like almost any Itallian food, but I can't handle _sea food_.
```
Comment (campoy) [inline talent/sourcerers/kuba--.md]: Itallian -> Italian

**Pull request merged**